avocado/core/runners/requirement_package.py

```diff
@@ -0,0 +1,133 @@
+import time
+from multiprocessing import Process, SimpleQueue
+
+from ...utils.software_manager.main import MESSAGES
+from ...utils.software_manager.manager import SoftwareManager
+from .. import nrunner
+
+
+class RequirementPackageRunner(nrunner.BaseRunner):
+    """Runner for requirements of type package
+
+    This runner handles, the installation, verification and removal of
+    packages using the avocado-software-manager.
+
+    Runnable attributes usage:
+
+     * kind: 'requirement-package'
+
+     * uri: not used
+
+     * args: not used
+
+     * kwargs:
+        - name: the package name (required)
+        - action: one of 'install', 'check', or 'remove' (optional, defaults
+          to 'install')
+    """
+
+    @staticmethod
+    def _run_software_manager(cmd, package, queue):
```

Comment:
Nitpick: Cognitive Complexity of 22.  This would be the only method that I would split into small ones, one for each action/command.